packages/codegen-ui-react/lib/utils/forms/validation.ts

```diff
@@ -0,0 +1,104 @@
+        };
+      case ValidationTypes.END_WITH:
+        return {
+          hasError: !validation.values.some((el) => value.endsWith(el)),
+          errorMessage: validation.validationMessage || `The value must end with ${validation.values.join(', ')}`,
+        };
+      case ValidationTypes.CONTAINS:
+        return {
+          hasError: !validation.values.some((el) => value.contains(el)),
+          errorMessage: validation.validationMessage || `The value must contain ${validation.values.join(', ')}`,
+        };
+      case ValidationTypes.NOT_CONTAINS:
+        return {
+          hasError: !validation.values.some((el) => !value.contains(el)),
+          errorMessage: validation.validationMessage || `The value must not contain ${validation.values.join(', ')}`,
+        };
+      case ValidationTypes.LESS_THAN_CHAR_LENGTH:
+        return {
+          hasError: !(value.length < validation.values),
+          errorMessage: validation.validationMessage || `The value must start with ${validation.values}`,
+        };
+      case ValidationTypes.GREATER_THAN_CHAR_LENGTH:
+        return {
+          hasError: !(value.length > validation.values),
+          errorMessage: validation.validationMessage || `The value must start with ${validation.values}`,
+        };
+      case ValidationTypes.LESS_THAN_NUM:
+        return {
+          hasError: !(value < validation.values),
+          errorMessage: validation.validationMessage || `The value must start with ${validation.values}`,
+        };
+      case ValidationTypes.GREATER_THAN_NUM:
+        return {
+          hasError: !(value < validation.values),
+          errorMessage: validation.validationMessage || `The value must start with ${validation.values}`,
+        };
+      case ValidationTypes.EQUAL_TO_NUM:
+        if (Array.isArray(validation.values)) {
+          return {
+            hasError: !validation.values.some((el) => el === value),
+            errorMessage: validation.validationMessage || `The value must start with ${validation.values.join(', ')}`,
+          };
+        }
+        return {
+          hasError: !(value === validation.values),
+          errorMessage: validation.validationMessage || `The value must start with ${validation.values}`,
+        };
+      case ValidationTypes.BE_AFTER:
+        return {
+          hasError: !(new Date(validation.values) > new Date(value)),
+          errorMessage: validation.validationMessage || `The value must start with ${validation.values}`,
+        };
+      case ValidationTypes.BE_BEFORE:
+        return {
+          hasError: !(new Date(validation.values) < new Date(value)),
+          errorMessage: validation.validationMessage || `The value must start with ${validation.values}`,
+        };
+      case ValidationTypes.EMAIL:
+        const EMAIL_ADDRESS_REGEX =
+          /^[-!#$%&'*+\/0-9=?A-Z^_a-z`{|}~](\.?[-!#$%&'*+\/0-9=?A-Z^_a-z`{|}~])*@[a-zA-Z0-9](-*\.?[a-zA-Z0-9])*\.[a-zA-Z](-?[a-zA-Z0-9])+$/;
```

Comment:
For Email, IpAddress validators, I used regex because we don't want to add dependencies